packages/react-aria-components/src/useDragAndDrop.tsx

```diff
@@ -135,6 +148,7 @@ export function useDragAndDrop(options: DragAndDropOptions): DragAndDrop {
       hooks.dropTargetDelegate = dropTargetDelegate;
       hooks.ListDropTargetDelegate = ListDropTargetDelegate;
     }
+    hooks.isDisabled = isDisabled;
```

Comment:
Can get rid of this as well if we rely on the state objects to tell us if drag or drop is disabled